spec/components/works/dates_component_spec.rb

```diff
@@ -50,4 +50,26 @@
       expect(rendered.css('#work_created_day option[@selected="selected"]').first['value']).to eq '8'
     end
   end
+
+  context 'with a populated form containing only year for single creation_date' do
+    let(:year_only_creation_date) { EDTF.parse('2020') }
+    let(:work) { build(:work, created_edtf: year_only_creation_date) }
+
+    it 'renders the component without month or day selected' do
+      expect(rendered.css('#work_created_year').first['value']).to eq '2020'
+      expect(rendered.css('#work_created_month option[@selected="selected"]')).to match_array([])
```

Comment:
Given it's an empty array in the expectation, I wonder how you'd feel about replacing `match_array([])` with `be_empty`

Reply:
Like it! Changed.